scripts/makeLegacyPlots.py

```diff
@@ -0,0 +1,1053 @@
+    hEWK       = getHisto("EWK", hBkgs, doOverflow)
+    hsingleT   = getHisto("singleT", hBkgs, doOverflow)
+    hTW        = getHisto("TW", hBkgs, doOverflow)
+    hZH        = getHisto("ZH", hBkgs, doOverflow)
+    hWH        = getHisto("WH", hBkgs, doOverflow)
+    hVV        = getHisto("VV", hBkgs, doOverflow)
+    httH       = getHisto("ttH", hBkgs, doOverflow)
+    hTTX       = getHisto("TTX", hBkgs, doOverflow)
+    hggH       = getHisto("ggH", hBkgs, doOverflow)
+    hVBFH      = getHisto("VBFH", hBkgs, doOverflow)
+    hVVV       = getHisto("VVV", hBkgs, doOverflow)
+    
+    hDYlist = [hDY_LM, hDY_0b_1Pt, hDY_0b_2Pt, hDY_0b_3Pt, hDY_0b_4Pt, hDY_0b_5Pt, hDY_0b_6Pt, hDY_1b_1Pt, hDY_1b_2Pt, hDY_1b_3Pt, hDY_1b_4Pt, hDY_1b_5Pt, hDY_1b_6Pt, hDY_2b_1Pt, hDY_2b_2Pt, hDY_2b_3Pt, hDY_2b_4Pt, hDY_2b_5Pt, hDY_2b_6Pt]
+    hothersList = [hEWK, hsingleT, hTW, hZH, hWH, hVV, httH, hTTX, hggH, hVBFH, hVVV]
+    hDY = makeSum("DY",hDYlist)
+    hothers = makeSum("other",hothersList)
+
+    hBkgList = [hothers, hWJets, hTT, hDY] # list for stack
+    hBkgNameList = ["Others", "W + jets", "t#bar{t}", "DY + jets"] # list for legend
+
+    if doQCD:
+        hQCD = getHisto ("QCD", hBkgs,doOverflow)
+        hBkgList.append(hQCD)
+        hBkgNameList.append("QCD")
+
+    hData = getHisto("data_obs", hDatas , doOverflow)
+
+    # remove all data from blinding region before doing anything else
+    if args.blindrange:
+        blow = float (args.blindrange[0]) 
+        bup = float (args.blindrange[1]) 
+        for ibin in range (1, hData.GetNbinsX()+1):
+            center = hData.GetBinCenter(ibin)
+            if center > blow and center < bup:
+                hData.SetBinContent(ibin, 0)
+
+    # these tw lists are used for the calculation of the syst error band. They must:
+    #       1. be in the same order
+    #       2. use the same names as those in the mainCfg (for hSystBkgNameList)
+    hSystBkgList = [hDY_LM, hDY_0b_1Pt, hDY_0b_2Pt, hDY_0b_3Pt, hDY_0b_4Pt, hDY_0b_5Pt, hDY_0b_6Pt, hDY_1b_1Pt, hDY_1b_2Pt, hDY_1b_3Pt, hDY_1b_4Pt, hDY_1b_5Pt, hDY_1b_6Pt, hDY_2b_1Pt, hDY_2b_2Pt, hDY_2b_3Pt, hDY_2b_4Pt, hDY_2b_5Pt, hDY_2b_6Pt, hTT, hWJets, hEWK, hsingleT, hTW, hZH, hWH, hVV, httH, hTTX, hggH, hVBFH, hVVV, hQCD]
+    hSystBkgNameList = ['DY_LM', 'DY_0b_1Pt', 'DY_0b_2Pt', 'DY_0b_3Pt', 'DY_0b_4Pt', 'DY_0b_5Pt', 'DY_0b_6Pt', 'DY_1b_1Pt', 'DY_1b_2Pt', 'DY_1b_3Pt', 'DY_1b_4Pt', 'DY_1b_5Pt', 'DY_1b_6Pt', 'DY_2b_1Pt', 'DY_2b_2Pt', 'DY_2b_3Pt', 'DY_2b_4Pt', 'DY_2b_5Pt', 'DY_2b_6Pt', 'TT', 'W', 'EWK', 'singleT', 'TW', 'ZH', 'WH', 'VV', 'ttH', 'TTX', 'ggH', 'VBFH', 'VVV', 'QCD']
+    hShapesNameList = ['trigSFDM0Up', 'trigSFDM1Up', 'trigSFDM10Up', 'trigSFDM11Up', 'trigSFDM0Down', 'trigSFDM1Down', 'trigSFDM10Down', 'trigSFDM11Down', 'etauFR_barrelUp', 'etauFR_endcapUp', 'PUjetIDSFUp', 'etauFR_barrelDown', 'etauFR_endcapDown', 'PUjetIDSFDown']
+    # the ETau, MuTau, and TauTau channels have some different shapes -> we add them here separately
```

Comment:
I have another comment: in ETau 2016 we don't have tau triggers,  so the only uncertainty should be "trigSFeleUp/Down"

Reply:
Yes, this is a thing I kept postponing: I have to separate the systematic sources also for the year, not only for channel. Now I do it.